src/map_extras.cpp

```diff
@@ -2625,6 +2625,39 @@ static void mx_looters( map &m, const tripoint &abs_sub )
     }
 }
 
+static void mx_corpses( map &m, const tripoint &abs_sub )
+{
+    const int num_corpses = rng( 1, 5 );
+    const auto gibs = item_group::items_from( "remains_human_generic", calendar::start_of_cataclysm );
+    //Spawn up to 5 human corpses in random places
+    for( int i = 0; i < num_corpses; i++ ) {
+        const tripoint corpse_location = { rng( 1, SEEX * 2 - 1 ), rng( 1, SEEY * 2 - 1 ), abs_sub.z };
+        if( g->is_empty( corpse_location ) ) {
+            m.add_field( corpse_location, fd_blood, rng( 1, 3 ) );
+            m.add_corpse( corpse_location );
+            //50% chance to spawn blood in every tile around every corpse in 1-tile radius
+            for( const auto &loc : m.points_in_radius( corpse_location, 1 ) ) {
+                if( one_in( 2 ) ) {
+                    m.add_field( { loc.xy(), abs_sub.z }, fd_blood, rng( 1, 3 ) );
```

Comment:
Why can't you just use `loc` (which happens to be a `tripoint`) directly? It should already have the same z-value as the input point (`corpse_location`), which is the same value as the one used here to create a new tripoint.
```suggestion
                    m.add_field( loc, fd_blood, rng( 1, 3 ) );
```

Reply:
Will fix this shortly, thanks for the review.